plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -545,12 +567,15 @@ static int prepare_dns_reply(u_char *data, const char *name, int type, int *dns_
          len = 12 + IP_ADDR_LEN;
          SAFE_CALLOC(answer, len, sizeof(u_char));
          p = answer;
-      
+
+         /* make TTL big-endian to work with DNS */
+         ttl_bige = htonl(ttl);
```

Comment:
I don't really see the need for the *ttl_bige* variable.
The host-byte-ordered value just need to be rewritten to the network-byte-ordered value. Not big- or little-endian
This can be done be reusing the variable: 
```c
ttl = htonl(ttl);
```